src/User/Infra/Security/PasswordEncoder.php

```diff
@@ -21,13 +21,15 @@ public function __construct(BasePasswordEncoder $encoder, string $salt = null)
         $this->salt = $salt ?? bin2hex(random_bytes(16));
```

Comment:
Minor food for thought, this will give the same salt to everyone in the same request/run, is that intentional?

Reply:
yes, more or less. Now we have `PasswordAlgorithm::$salt` (passed to hash/isValid) this would be the default value if $salt is null, [as SF requires this arg to be string](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Core/Encoder/PasswordEncoderInterface.php#L25). SF decides to use/ignore it from there per implem.

Reply:
What's bugging me now.. is we're going to use _only_ `PassAlgo::$salt` and silently ignore other settings like `PassAlgo::$type` and such SF does not support those at this point. Not sure we care really :)